src/main/java/com/redhat/devtools/lsp4ij/features/documentation/LSPDocumentationHelper.java

```diff
@@ -38,8 +39,9 @@ public static String convertToHTML(@Nullable MarkupContent content,
         if (content == null) {
             return "";
         }
+        Project project = editor.getProject();
         String htmlBody = MarkupKind.MARKDOWN.equals(content.getKind()) ?
-                MarkdownConverter.toHTML(content.getValue()) :
+                MarkdownConverter.getInstance(project).toHTML(content.getValue()) :

```

Comment:
project might be null

Reply:
Not true with last code.

Reply:
Code has changed, this comments is not relevant now